src/response/records.ts

```diff
@@ -361,6 +361,35 @@ export interface Disposition {
   readonly ruleLabel?: string;
 }
 
+/**
+ * This object describes one of the reasons for the multiplier.
+ */
+export interface Reason {
+  /**
+   * The machine-readable code for the reason.
+   */
+  code: string;
+  /**
+   * The human-readable description of the reason.
+   */
+  reason: string;
+}
+
+/**
+ * The object describing the risk score multiplier and the reasons for that
+ * multiplier.
+ */
+export interface RiskScoreReason {
+  /**
+   * The risk score multiplier.
```

Comment:
We should update this (and the other outputs) to match the [dev site description](https://github.com/maxmind/dev-site/pull/1177/files#diff-9ebe00180cb4792b562b8944a7493af7a30336924eae359bc5f4f423c69014b9R21.